lib/sisimai/bite/email/kddi.rb

```diff
@@ -0,0 +1,166 @@
+module Sisimai::Bite::Email
+  module KDDI
+    # Sisimai::Bite::Email::KDDI parses a bounce email which created by au by KDDI.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/KDDI.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'       => %r/no-reply[@].+[.]dion[.]ne[.]jp/,
+        :'reply-to'   => %r/\Afrom[ \t]+\w+[.]auone[-]net[.]jp[ \t]/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->